games/sudoku-solver/Sudoku.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+class Sudoku {
+  public void sudokuSolver(int[][] board) {
+    if(board.length == 0) return;
+    sudokuSolverHelper(board);
+    printBoard(board);
```

Comment:
Do not print board as part of this function. Expose this function outside and call it directly from main.

Reply:
Agree, fixed here e18c92a